### PR TITLE
streamline get-started instructions

### PR DESCRIPTION
**Pull request by mattf**

### CODE REVIEW

**Comment by mattf** [comment]
@elmiko @thesteve0 ptal

File: get-started.md [hunk 1]
```diff
@@ -50,38 +50,28 @@ directly from a source repository.
...
+First, install all the Oshinko resources into your project -
 
 ```bash
-: Load all the Oshinko resources into your OpenShift project
 oc create -f http://radanalytics.io/resources.yaml
+```
+
+This creates all the Oshinko S2I (source-to-image) templates, the Oshinko Web UI application, as well as a ServiceAccount and RoleBinding needed for creation and management of Apache Spark clusters.
+
+Second, setup the Oshinko Web UI application -
```
Comment (elmiko) [inline get-started.md]: i think we should leave some of the "launch" language here, what about changing this to:

    Second, start the Oshinko Web UI application -

Comment (mattf) [inline get-started.md]: updated

**Comment by elmiko** [comment]
thanks Matt!

**Pull request merged**